requirements/requirements-test.txt

```diff
@@ -1,7 +1,7 @@
 -r requirements.txt
 
-pytest==3.4.0
 coverage<4.4
-pytest-cov==2.4.0
+pytest>=3.4.0
+pytest-cov>=2.4.0
 codeclimate-test-reporter==0.2.3
```

Comment:
The main issue is here, the project should move to https://github.com/codeclimate/test-reporter. The python version is deprecated. Whatever that dependencies order fix the CI.